.markdownlint.yml

```diff
@@ -0,0 +1,19 @@
+# SPDX-FileCopyrightText: Copyright (c) 2016-2025 Objectionary.com
+# SPDX-License-Identifier: MIT
+---
+# Default state for all rules
+default: true
+
+# Line length : https://github.com/DavidAnson/markdownlint/blob/v0.33.0/doc/md013.md
+MD013:
+  line_length: 100
+
+# Inline HTML : https://github.com/DavidAnson/markdownlint/blob/v0.33.0/doc/md033.md
+MD033:
+  allowed_elements: [ img ]
+
+# Fenced code blocks should have a language specified : https://github.com/DavidAnson/markdownlint/blob/v0.33.0/doc/md040.md
+MD040: false
```

Comment:
@volodya-lombrozo what's the point of disabling this? Just use `text` where you don't know the language.

Reply:
@yegor256 Sure. Can't agree more.